src/main/java/com/dougnoel/sentinel/steps/TableSteps.java

```diff
@@ -145,4 +147,25 @@ public static void storeTableValues(int pageNumber, String tableName) {
     public static void clickColumnHeaderToSort(String columnName, String tableName) {
     	getElementAsTable(tableName).clickColumnHeader(columnName);
     }
+
+	/**
+	 * Clicks the xpath of element in row which have stored value in column name
+	 * <p>
+	 * <b>Gherkin Examples:</b>
+	 * <ul>
+	 * <li>I find the Name column in the Project Files Table in row with value entered for the Search Input and click the xpath //i[contains(concat(' ', @class, ' '), ' fa-toggle-on ')]</li>
+	 * </ul>
+	 * @param columnName String the name of the column to click the header of
+	 * @param tableName String the name of the table element
+	 * @param key String the key used to retrieve the value
+	 * @param xpath String xpath value to element to click
+	 */
+	@When("^I find the (.*?) column in the (.*?) in row with value (?:entered|selected|used) for the (.*?) and click the xpath (.*?)$")
+	public static void clickStoredTextRowContainsXpath(String columnName, String tableName, String key, String xpath) {
+		var textToMatch = Configuration.toString(key);
+		String errorMessage = SentinelStringUtils.format("No previously stored text was found for the \"{}\" key.", key);
+		Assert.assertNotNull(errorMessage, textToMatch);
```

Comment:
This should be done as `assertNotNull(errorMessage, testToMatch);` And include should be
`import static org.junit.Assert.assertNotNull;` However, this is not a test so this assertion shouldn't be used here. (See my other comment.)